Extension/src/LanguageServer/copilotProviders.ts

```diff
@@ -38,22 +39,54 @@ export async function registerRelatedFilesProvider(): Promise<void> {
 
                         const getIncludesHandler = async () => (await getIncludesWithCancellation(1, token))?.includedFiles.map(file => vscode.Uri.file(file)) ?? [];
                         const getTraitsHandler = async () => {
-                            const chatContext: ChatContextResult | undefined = await (getActiveClient().getChatContext(token) ?? undefined);
+                            const cppContext = await getProjectContext(context, token);
```

Comment:
@kuchungmsft Can you pass the uri instead of relying on the active file on the server? I don't think they can be relied on to match all the time. Same for the pre-existing getIncludes.

Reply:
Fixed, thanks.